src/controller/src/commands/interrupt_command_command.cpp

```diff
@@ -0,0 +1,18 @@
+#include <goliath/controller/commands/interrupt_command_command.h>
+
+using namespace goliath::commands;
+using namespace goliath;
+
+InterruptCommandCommand::InterruptCommandCommand(const size_t &id, std::shared_ptr<CommandMap> &commandMap)
+    : BasicCommand(id, {}), commandMap(commandMap) {
+}
+
+void InterruptCommandCommand::execute(handles::HandleMap &handles, const proto::CommandMessage &message) {
+    if ((*commandMap).commandExists(static_cast<size_t>(message.interruptcommandcommand().commandid()))) {
```

Comment:
volgens mij kan je hier `->` gebruiken.
Daarnaast is het misschien verstandig om hier: https://github.com/P8P-7/proto/compare/861e2d64505d0742f9a94d31f774cd7925fbce47...dda9a8ae8684279e82abf58247d5046e303567b4#diff-334f17347257bc177740eea219a1a61cR8 een `uint64`  te gebruiken voor commandId. Dan komt het geloof ik overeen met `size_t` in C++ / GCC. Dan gaat deze cast altijd goed. Anders moet je misschien eventjes checken of het binnen de bounds valt.

Reply:
Operator `->` werkt niet op een `operator[]`